### PR TITLE
Do not pass token as argument

### PR DESCRIPTION
**Pull request by FrNecas**
The script expects only version and path to specfile, not token. The token is obtained from `INPUT_` env var, therefore we shouldn't pass it in.

### CODE REVIEW

**Review by nforro — APPROVED** [review]

**Comment by jpopelka** [comment]
tl;dr; please revert :-)

In the chat, you described this as "a bug", what was the buggy behavior?

> The script expects only version and path to specfile, not token.

Yes, the `token` is not required, but could be set if one wanted to use a different token than the [default](https://github.com/packit/prepare-release/blob/main/action.yml#L12), [autogenerated one](https://docs.github.com/en/actions/security-guides/automatic-token-authentication).
That would be needed if this action would be expected to trigger another action in a workflow, [in which case the autogenerated token wouldn't work](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow).

> The token is obtained from `INPUT_` env var

and [the `INPUT_TOKEN` is created from the `token` input](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs) ( [defaults](https://github.com/packit/prepare-release/blob/main/action.yml#L12) to the [autogenerated token](https://docs.github.com/en/actions/security-guides/automatic-token-authentication)).

**Comment by FrNecas** [comment]
@jpopelka Check the action logs here https://github.com/packit/packit/actions/runs/4083839207/jobs/7039829912 , passing the token to the script like it was in the action did not work (unexpected argument). Now the action seems to work fine.

**Comment by FrNecas** [comment]
I think that passing in `token` as input will still work fine, because that will be applied as `INPUT_TOKEN` variable, therefore picked up by the script. I merely removed the token from arguments of `prepare-release.py` which doesn't expect it.

**Comment by jpopelka** [comment]
I see now. Thank you and sorry for the fuss.
